lib/parse.js

```diff
@@ -93,7 +112,7 @@ module.exports = function (input) {
             token = { type: 'function', sourceIndex: pos - name.length, value: name, before: value.slice(pos + 1, next) };
             pos = next;
 
-            if (name === 'url' && code !== singleQuote && code !== doubleQuote) {
+            if (name === 'url' && code !== singleQuote && code !== doubleQuote && !(code === slash && value.charCodeAt(pos + 1) === star)) {
```

Comment:
I just don't like ! before parentheses. More like De Morgan's laws in such cases.